.travis.yml

```diff
@@ -42,6 +42,7 @@ deploy:
   provider: pypi
   user: okfn
   distributions: sdist bdist_wheel
+  skip_cleanup: true
```

Comment:
Presumably this fixes the formatting issue on pypi. Does this need adding to the example python repository? https://github.com/okfn/oki-py

Reply:
Yes. Added a PR - https://github.com/okfn/oki-py/pull/11